apps/condo/domains/acquiring/schema/RegisterMultiPaymentService.js

```diff
@@ -453,12 +453,16 @@ const RegisterMultiPaymentService = new GQLCustomSchema('RegisterMultiPaymentSer
                 if (deletedReceiptsIds.length) {
                     throw new GQLError({ ...ERRORS.RECEIPTS_ARE_DELETED, messageInterpolation: { ids: deletedReceiptsIds.join(', ') } }, context)
                 }
-
-                const negativeReceiptsIds = receipts
-                    .filter(receipt => Big(receipt.toPay).lte(0))
-                    .map(receipt => receipt.id)
-                if (negativeReceiptsIds.length) {
-                    throw new GQLError({ ...ERRORS.RECEIPTS_HAVE_NEGATIVE_TO_PAY_VALUE, messageInterpolation: { ids: negativeReceiptsIds.join(', ') } }, context)
+                //If we have receipt with overpayment, we still want to allow user to pay partially for it
+                //From previous checks we know that every receipt should have distribution amount otherwise we got err at previous step
+                //If distribution is set then payment amount will be taken from distribution not from receipt toPay
+                if (distributionReceipts.length === 0){
```

Comment:
distributionReceipts can be negative
'input RegisterMultiPaymentReceiptAmountInput { receipt: BillingReceiptWhereUniqueInput!, amount: String! }'